pkg/sessions/tests/mock_store.go

```diff
@@ -46,12 +50,22 @@ func (s *MockStore) Load(_ context.Context, key string) ([]byte, error) {
 	return entry.data, nil
 }
 
+// Releases a previously set lock
+func (s *MockStore) ReleaseLock(_ context.Context, key string) error {
+	delete(s.locksCache, key)
+	return nil
+}
+
 // Clear deletes an entry from the memory cache
 func (s *MockStore) Clear(_ context.Context, key string) error {
 	delete(s.cache, key)
 	return nil
 }
 
+func (s *MockStore) Lock(key string) sessions.Lock {
+	return &sessions.NoOpLock{}
+}
```

Comment:
This doesn't seem to be the opposite of release lock? Shouldn't there be a MockLock in use here? Or do we somehow need to do something with the lock cache?